shared/fs/common/path-item-action-container.js

```diff
@@ -118,14 +98,30 @@ const getRootMenuItems = (stateProps, dispatchProps) => {
   return menuItems
 }
 
-const mergeProps = (stateProps, dispatchProps) => {
-  const {path, pathItem, _username, childrenFolders, childrenFiles} = stateProps
-  const {onHidden, _loadMimeType} = dispatchProps
+const mergeProps = (stateProps, dispatchProps, {path, actionIconClassName, actionIconFontSize}) => {
+  const {_pathItems, _username} = stateProps
+  const {_loadMimeType} = dispatchProps
+  const pathItem = _pathItems.get(path) || Constants.makeUnknownPathItem()
+  const [childrenFolders, childrenFiles] =
```

Comment:
might be clearer to just use an object instead of an array-as-tuple, aka
const {folders, files} = pathItem.children.reduce((...) => {}, {folder: 0, files: 0})